kt/godot-library/build.gradle.kts

```diff
@@ -38,8 +38,10 @@ dependencies {
     // we need to add it here so reflection is available where the code is loaded (Bootstrap.kt) otherwise it will not work
     api(kotlin("reflect", version = libs.versions.kotlin.get()))
     implementation("com.utopia-rise:tools-common:$fullGodotKotlinJvmVersion")
+    testImplementation("junit", "junit", "4.12")
```

Comment:
Why junit 4 and not 5?

Reply:
Because I don't know how to use Junit 5 :p

Reply:
As discussed on discord. Fine for now.